superset-frontend/packages/superset-ui-core/src/color/CategoricalColorScale.ts

```diff
@@ -39,99 +38,172 @@ class CategoricalColorScale extends ExtensibleFunction {
 
   scale: ScaleOrdinal<{ toString(): string }, string>;
 
-  parentForcedColors: ColorsLookup;
-
   forcedColors: ColorsLookup;
 
+  sharedColorMapInstance: ReturnType<typeof getSharedLabelColor>;
+
+  sliceMap: Map<string, string>;
```

Comment:
I'm assuming it's a mapping of series/dimensions and colours assigned to them? If so, could we maybe change that name, since we're often calling saved charts "slices", which could be confusing here?

Reply:
Sure makes sense